documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -1,65 +1,30 @@
-== Installing over the Network (BETA)
+== Installing over the Network
 
-WARNING: Network installation is in currently in PUBLIC BETA. Please do not rely on it in production.
+WARNING: Network installation is in currently only available in the LATEST/STABLE bootloader build.
```

Comment:
Is this WARNING: block still necessary, given the similar IMPORTANT: block below?

Reply:
This is the one right at the top? If so, yes. I want something *right* at the start of the section to tell folks that their experience might be problematic. Thinking about it while "PUBLIC BETA" was fairly explicable, "LATEST/STABLE" is probably less well known. Might want to rephrase this somehow, but I do want a `WARNING` right at the start here.

Reply:
I've moved the IMPORTANT to the top and changed it to warning and tweaked the wording